package.json

```diff
@@ -19,22 +19,22 @@
     "babel-loader": "^7.1.2",
     "babel-preset-env": "^1.6.0",
     "jasmine": "2.4.1",
-    "jquery": "^2.2.4",
-    "jsdom": "^9.2.1",
+    "jquery": "^3.2.1",
+    "jsdom": "^11.5.1",
     "json-loader": "^0.5.4",
     "sinon": "^1.17.4",
-    "webpack": "^3.5.5",
-    "ws": "^1.1.0",
+    "webpack": "^3.10.0",
+    "ws": "^1.1.5",
     "xmlhttprequest": "^1.8.0"
   },
   "peerDependencies": {
-    "jquery": "^2.2.4"
+    "jquery": "^3.2.1"
   },
   "repository": {
     "type": "git",
     "url": "git://github.com/firehoseio/js_client.git"
   },
   "dependencies": {
-    "xhr": "^2.4.0"
+    "xhr": "^2.4.1"
```

Comment:
Is this really a dependency? Maybe it should be dev or peer

Reply:
There are to ways to consume this library:
1. The module way
2. Loading this up via script and something similar where its using the distributed compiled file

In option 1, I'll be including it as peer since nothing is compiled until the consumer compiles their app bundle. In option 2, `xhr` needs to be bundle to work as a standalone distributed compiled file/lib.

Reply:
Actually, leaving it as a dependency is OK in both scenarios. This is an effort to remove `jquery`.